### PR TITLE
Resprite critter gunbots

### PR DESCRIPTION
**Pull request by flappybatpal**
## About the PR 
Resprites gunbots with sprites by Matheus.
Adds glowing red eyes.
Adds animated treads.

## Why's this needed? 
They fit the new borgs better!

image

## Changelog

```changelog
(u)Flappybat and THISISANICEGAME 
(+)Resprited the critter security bot, now with animated treads and glowing eyes.
```

### CODE REVIEW

**Comment by flappybatpal** [comment]
Could not get the glowing eyes to work in the same way cyborg heads work. SELFILLUM is nearly the same but doesn't quite have the same change in different lighting. If anyone could work out the way to implement that it would be nice.

I copied the very simple implementation and sprites of Pali's PR and had 0 success.

**Comment by Flaborized** [comment]
Looks good and fits resprite 👍

File: code/mob/living/critter/gunbot.dm [hunk 1]
```diff
@@ -18,6 +18,12 @@
 	speechverb_ask = "queries"
 	metabolizes = 0
 
+	New()
+		. = ..()
+		var/image/eye_light = image(icon, "mars_sec_bot_eye")
+		eye_light.plane = PLANE_SELFILLUM
+		src.UpdateOverlays(eye_light, "eye_light")
```
Comment (ZeWaka) [inline code/mob/living/critter/gunbot.dm:25]: so you're saying this doesn't work?

Comment (flappybatpal) [reply code/mob/living/critter/gunbot.dm:25]: Oh no that works, it's just pretty different to the implementation on borgs which use black pixels for eyes and put them on lighting plane.

https://github.com/goonstation/goonstation/blob/697eb5d04de29942d325694dcba639db181a12b9/code/mob/living/silicon/robot.dm#L2590

**Review by virvatuli-gh — APPROVED** [review]

**Pull request merged**